### PR TITLE
Rename Android test app package to com.microsoft.reacttestapp

### PR DESCRIPTION
**Pull request by sweggersen**
Rename Android test app package from `com.react.testapp` to `com.microsoft.reacttestapp` to align with iOS.

### CODE REVIEW

**Review by arazabishov — APPROVED** [review]

File: android/app/src/androidTest/java/com/react/testapp/ManifestTest.kt [hunk 1]
```diff
@@ -1,8 +1,8 @@
-package com.react.testapp
+package com.microsoft.reacttestapp
```
Comment (tido64) [inline android/app/src/androidTest/java/com/react/testapp/ManifestTest.kt:1]: Do we need to rename the folders as well to match?

**Review by tido64 — APPROVED** [review]
Please make sure to prefix the commit message with `fix(Android): ` so it gets published.

File: android/app/src/androidTest/java/com/react/testapp/ManifestTest.kt (hunk 1 above)
Comment (tido64) [reply android/app/src/androidTest/java/com/react/testapp/ManifestTest.kt:1]: We need to rename `androidTest/java/com/react/testapp/` and `flipper/java/com/react/testapp/` as well

Comment (sweggersen) [reply android/app/src/androidTest/java/com/react/testapp/ManifestTest.kt:1]: Fixed

**Pull request merged**